.github/workflows/helm_chart.yaml

```diff
@@ -0,0 +1,20 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Helm Chart
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  push-chart:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: goodsmileduck/helm-push-action@v3.2.4
+      env:
+        SOURCE_DIR: '.'
+        CHART_FOLDER: '.'
+        CHARTMUSEUM_URL: 'https://charts.softonic.io'
+        CHARTMUSEUM_USER: '${{ secrets.CHARTMUSEUM_USER }}'
```

Comment:
where this secret comes from? Is it correct this naming? what if we have different charmuseum hubs?

Reply:
this is outdated.
In any case the secret comes from the repos' settings, so we can have one chartmuseum each repo.
If we had multiple charmuseums for one repo, we'll see how to tackle that